samples/crypto/aes_ctr/README.rst

```diff
@@ -0,0 +1,61 @@
+.. _crypto_aes_ctr:
+
+Crypto: AES CTR
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The AES sample shows how to perform AES encryption and decryption operations using the CTR block cipher mode without padding using a 128-bit AES key.
+
+First, the sample performs initalization:
+   #. The Platform Security Architecture (PSA) API is initialized.
+   #. A random AES key is generated and imported in the PSA crypto keystore.
+
+Then, the sample encrypts/decrypts a sample plaintext:
+   #. A random initialization vector (IV) is generated.
+   #. Encryption is performed.
+   #. Decryption is performed.
+
+Afterwards, the sample performs cleanup:
+   #. The AES key is removed from the PSA crypto keystore.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows:  nrf5340dk_nrf5340_cpuappns, nrf9160dk_nrf9160ns
```

Comment:
It seems some of these `README.rst` files have a different list of supported kits compared to the `testcase.yaml` files.

Reply:
@b-gent: Comment on this?

Reply:
``.. table-from-rows:: `` requires you to manually provide these boards.

There is an option of using `` :sample-yaml-rows: `` instead, then it should automatically include the right targets based in the yaml file.

https://github.com/nrfconnect/sdk-nrf/blob/master/doc/nrf/templates/sample_README.rst#requirements

Reply:
Fixed